src/pabot/result_merger.py

```diff
@@ -150,6 +150,8 @@ def merge_missing_tests(self, suite):
 
     def merge_time(self, suite):
         cur = self.current
+        if ROBOT_VERSION >= "7.0":
+            cur.elapsed_time = None
```

Comment:
There is also the legacy output format https://robotframework.org/robotframework/latest/RobotFrameworkUserGuide.html#legacy-output-file-format

I would limit this change to cases where `--legacyoutput` is not used.

Reply:
add the legacy_output option check